### ISSUE TITLE
Genericons not found

### ISSUE THREAD

**Issue by misulicus** [easydigitaldownloads/themedd#208]
original report from customer with a screenshot attached
````
Minor bug: The font-face rule in style.css has the incorrect URL for Genericons. 
I found this after running my site through deadlinkchecker.com
````
https://secure.helpscout.net/conversation/641172552/87929?folderId=180499

I am able to see the error in IE browser, its attempting to load https://edd.rmjsolutions.ro/wp-content/themes/themedd/fonts/Genericons.eot? which does not exists.

**Comment by davidsherlock** [comment]
This will be fixed in the 1.1 release of Themedd, which has switched over to Bootstrap and dropped the need for Genericons.

**Comment by amdrew** [comment]
Will get this fixed until 1.1 is released.
